src/languages/en.ts

```diff
@@ -2970,6 +2983,34 @@ export default {
         genericUpdateReportFieldFailureMessage: 'Unexpected error while updating the field, please try again later.',
         genericUpdateReporNameEditFailureMessage: 'Unexpected error while renaming the report, please try again later.',
         noActivityYet: 'No activity yet',
+        actions: {
+            type: {
+                changeField: ({oldValue, newValue, fieldName}: ChangeFieldParams) => `changed ${fieldName} from ${oldValue} to ${newValue}`,
+                changePolicy: ({fromPolicy, toPolicy}: ChangePolicyParams) => `changed policy from ${fromPolicy} to ${toPolicy}`,
+                changeType: ({oldType, newType}: ChangeTypeParams) => `changed type from ${oldType} to ${newType}`,
+                delegateSubmit: ({delegateUser, originalManager}: DelegateSubmitParams) => `sent this report to ${delegateUser} since ${originalManager} is on vacation`,
+                exportedToCSV: `exported this report to CSV`,
+                exportedToIntegration: ({label}: ExportedToIntegrationParams) => `exported this report to ${label}`,
+                forwarded: ({amount, currency}: ForwardedParams) => `approved ${currency}${amount}`,
+                // verify errorMessage
+                integrationsMessage: ({errorMessage}: IntegrationsMessageParams) => `failed to export this report to NetSuite. ${errorMessage}`,
```

Comment:
Just checking but `NetSuite` here doesn't seem to be variable depending on what accounting solution is connected? I.e they could be exporting to Xero, not NetSuite, for example.